Betable/Betable.m

```diff
@@ -42,11 +44,18 @@
 NSString *BetablePasteBoardName = @"com.Betable.BetableSDK.sharedData";
 
 #define SERVICE_KEY @"com.betable.SDK"
-#define USERNAME_KEY @"com.betable.AccessToken"
+#define USERNAME_KEY @"com.betable.Credentials"
 #define FIRSTSTORE_KEY @"com.betable.FirstStore"
 
+// int disguied as enum used in place of double disguied as NSTimeInterval.
+typedef enum heartbeatPeriods {
+    NOW = 0,
+    HEALTHY_PERIOD = 60,
+    UNHEALTHY_PERIOD = 5
+} HeartBeatPeriods;
+
 @interface Betable () {
-    NSMutableArray *_deferredRequests;
+    NSMutableOrderedSet  *_deferredRequests;
```

Comment:
This was only for loading of the profile. I don't think we need to defer any requests anymore, but I could be wrong.